content/guides/genai-claude-code-mcp/claude-code-mcp-guide.md

```diff
@@ -1,387 +1,321 @@
+---
+# Use Claude Code and Docker MCP Toolkit to generate a Docker Compose file from natural language
 
-## Why Claude Code and Docker MCP Toolkit work well together
 
-MCP gives you the protocol, but the Docker MCP Toolkit makes it practical. Running MCP servers manually means handling Node versions, Python dependencies, scattered credentials, and different setups across machines. A simple setup often turns into hours of fixing environment issues.
+- **Setup**: Enable Docker MCP Toolkit → Add Docker Hub MCP server → Connect Claude Code
+- **Use Claude**: Describe your stack in natural language to Claude Code
+- **Automate**: Claude searches Docker Hub via MCP and generates production-ready `docker-compose.yml`
+- **Deploy**: Run `docker compose up` and your Node.js + PostgreSQL app is live on `localhost:3000`
+- **Key Benefit**: No manual YAML writing, no image hunting — describe once, Claude builds it
 
-Docker MCP Toolkit removes all that frustration.
+**Time to completion**: ~15 minutes
 
-You get:
+---
 
-- More than 200 ready-to-use MCP servers  
-- One-click launch inside Docker Desktop  
-- Secure credential handling  
-- The same setup on Mac, Windows, and Linux  
-- Automatic updates when new versions arrive  
+## 1. What you are building
 
-If you already use Docker Desktop, this feels natural. Connecting Claude Code to your tools becomes quick and painless.
+The goal is simple: use Claude Code together with the Docker MCP Toolkit to search Docker Hub images and generate a complete docker compose file for a Node.js and PostgreSQL setup. 
 
+The Model Context Protocol (MCP) bridges Claude Code and Docker Desktop, giving Claude real-time access to Docker's tools. Instead of context-switching between Docker, terminal commands, and YAML editors, you describe your requirements once and Claude handles the infrastructure details.
 
-## Setting up Claude Code with Docker MCP Toolkit
+**Why this matters:** This pattern scales to complex multi-service setups, database migrations, networking, security policies — all through conversational prompts.
 
-### Prerequisites
+---
 
-- [Install Docker Desktop](https://www.docker.com/products/docker-desktop/) 4.40 or later  
-- [MCP Toolkit enabled](https://docs.docker.com/ai/mcp-catalog-and-toolkit/get-started/?_gl=1*e0exvl*_gcl_au*MjEyODc5MTk5NS4xNzU3NTQ5MTA5*_ga*OTQ5NjAzMjcyLjE3NTc1NDkwNjQ.*_ga_XJWPQMJYHQ*czE3NjMwNzEyODMkbzEyJGcxJHQxNzYzMDcxMzY3JGo2MCRsMCRoMA..#enable-docker-mcp-toolkit)  
+## 2. Prerequisites
 
-## Step 1. Install Claude Code
+Make sure you have:
 
-```bash
-# Install Claude Code
-curl -fsSL https://claude.ai/install.sh | sh
+- Docker Desktop installed
+- Docker Desktop updated with MCP Toolkit support
```

Comment:
Link to MCP toolkit setup as a prerequisite and remove it from the guide steps: https://docs.docker.com/ai/mcp-catalog-and-toolkit/get-started/#setup